services/spd/trusty/trusty.c

```diff
@@ -99,6 +99,10 @@ static struct args trusty_context_switch(uint32_t security_state, uint64_t r0,
 	ret.r1 = r1;
 	ret.r0 = r0;
 
+#if CTX_INCLUDE_FPREGS
```

Comment:
Looks good, although It might be nice to add a comment here to say FP&SIMD state should be preserved by the SMC caller in the CPU suspend/resume case.

Reply:
Thanks, uploaded new.